lib/puppet/provider/reboot/linux.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../reboot/posix'
```

Comment:
According to metadata.json, it supports puppet >=3.3.0. I will find out what that bundles.

It didn't work with './posix', but worked with one folder up.